pkg/errors/errors.go

```diff
@@ -87,20 +87,20 @@ func Wrap(err error, next error) error {
 		return fmt.Errorf("%s: %w", err.Error(), next)
 	}
 
-	err = nil
+	var forw error
```

Comment:
@henrybarreto, as we discussed, this PR isn't intended to fix the lints but to allow the new linter to be used and mark the lints for future work. So please avoid this unrelated change in this PR.